src/diff/index.js

```diff
@@ -232,6 +232,12 @@ export function diff(
 
 		if ((tmp = options.diffed)) tmp(newVNode);
 	} catch (e) {
+		if (isHydrating && excessDomChildren != null) {
```

Comment:
Want to add a comment here why we do this? I know myself good enough to know I forget tomorrow :D